### DISCUSSION TITLE
How to Create Cumulative Time Series in WebMapServiceCatalogItem for TerriaJS?

### DISCUSSION THREAD

**Discussion by rnnchy** [TerriaJS/terriajs#7254] [Q&A]
Hi folks! Is there a way to make the time series in a WebMapServiceCatalogItem cumulative? I want it to also display the previous date

**Comment by mwu2018** [comment]
You might use the time query parameter. See the documentation at https://docs.geoserver.org/2.23.x/en/user/services/wms/time.html.
E.g., query for the 16 hours period before 11pm, 2014-01-01 https://nationalmap.gov.au/proxy/_0d/https://gis.aremi.data61.io/bom/wms?transparent=true&format=image%2Fpng&exceptions=XML&styles=solar_satellite&version=1.3.0&time=PT16H/2014-01-01T23%3A00%3A00Z&service=WMS&request=GetMap&layers=Solar_Satellite_DNI_2014&bbox=10018754.17139462%2C-10018754.17139462%2C20037508.342789244%2C0&width=256&height=256&crs=EPSG%3A3857

For ArcGisMapServerCatalogItem, there are some time related traits that can be used. See https://docs.terria.io/guide/connecting-to-data/catalog-type-details/esri-mapserver/

**Comment by mwu2018** [comment]
In the future, a time window query similar to ArcGisMapServerCatalogItem might be added to WebMapServiceCatalogIte.
